src/User32.Desktop/User32.Helpers.cs

```diff
@@ -32,6 +32,130 @@ public static SafeDCHandle GetDC(IntPtr hWnd)
+        /// existing message-only window.
+        /// </para>
+        /// </param>
+        /// <param name="hMenu">
+        /// Handle to a menu, or specifies a child-window identifier, depending on the window style.
+        /// For an overlapped or pop-up window, hMenu identifies the menu to be used with the window;
+        /// it can be NULL if the class menu is to be used. For a child window, hMenu specifies the
+        /// child-window identifier, an integer value used by a dialog box control to notify its
+        /// parent about events. The application determines the child-window identifier; it must be
+        /// unique for all child windows with the same parent window.
+        /// </param>
+        /// <param name="hInstance">
+        /// Handle to the instance of the module to be associated with the window.
+        /// </param>
+        /// <param name="lpParam">
+        /// Pointer to a value to be passed to the window through the CREATESTRUCT structure
+        /// (lpCreateParams member) pointed to by the lParam param of the WM_CREATE message. This
+        /// message is sent to the created window by this function before it returns.
+        /// <para>
+        /// If an application calls CreateWindow to create a MDI client window, lpParam should point
+        /// to a CLIENTCREATESTRUCT structure. If an MDI client window calls CreateWindow to create
+        /// an MDI child window, lpParam should point to a MDICREATESTRUCT structure. lpParam may be
+        /// NULL if no additional data is needed.
+        /// </para>
+        /// </param>
+        /// <returns>
+        /// If the function succeeds, the return value is a handle to the new window.
+        /// <para>
+        /// If the function fails, the return value is NULL. To get extended error information, call GetLastError.
+        /// </para>
+        /// <para>This function typically fails for one of the following reasons:</para>
+        /// <list type="">
+        /// <item>an invalid parameter value</item>
+        /// <item>the system class was registered by a different module</item>
+        /// <item>The WH_CBT hook is installed and returns a failure code</item>
+        /// <item>
+        /// if one of the controls in the dialog template is not registered, or its window window
+        /// procedure fails WM_CREATE or WM_NCCREATE
+        /// </item>
+        /// </list>
+        /// </returns>
+        public static IntPtr CreateWindow(
+           string lpClassName,
+           string lpWindowName,
+           WindowStyles dwStyle,
+           int x,
+           int y,
+           int nWidth,
+           int nHeight,
+           IntPtr hWndParent,
+           IntPtr hMenu,
+           IntPtr hInstance,
+           IntPtr lpParam)
+        {
+            return CreateWindowEx(0, lpClassName, lpWindowName, dwStyle, x, y, nWidth, nHeight, hWndParent, hMenu, hInstance, lpParam);
```

Comment:
This is a mapping from the API only, On C/C++ this is a macro and this is documented on the [MSDN](https://msdn.microsoft.com/en-us/library/ms632679%28VS.85%29.aspx?f=255&MSPPError=-2147217396): 

>**CreateWindow** is implemented as a call to the **CreateWindowEx** function, as shown below.

```
#define CreateWindowA(lpClassName, lpWindowName, dwStyle, x, y, nWidth, nHeight, hWndParent, hMenu, hInstance, lpParam)\
CreateWindowExA(0L, lpClassName, lpWindowName, dwStyle, x, y, nWidth, nHeight, hWndParent, hMenu, hInstance, lpParam)

#define CreateWindowW(lpClassName, lpWindowName, dwStyle, x, y, nWidth, nHeight, hWndParent, hMenu, hInstance, lpParam)\
CreateWindowExW(0L, lpClassName, lpWindowName, dwStyle, x, y, nWidth, nHeight, hWndParent, hMenu, hInstance, lpParam)

#ifdef UNICODE
#define CreateWindow  CreateWindowW
#else
#define CreateWindow  CreateWindowA
#endif
```